src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -89,9 +112,60 @@ def drop_application(self, auto_yes: bool):
                     cc.message(f"Did not drop application object {self.app_name}.")
                     return  # The user desires to keep the app, therefore exit gracefully
 
-        # 4. All validations have passed, drop object
+        # 4. Check for application objects owned by the application
+        application_objects = self.get_application_objects()
+        if len(application_objects) > 0:
+            application_objects_str = self._application_objects_to_str(
+                application_objects
+            )
+            if cascade is True:
+                cc.message(
+                    dedent(
+                        f"""\
+                            The following application objects will be deleted:
```

Comment:
How about "The following objects are owned by application <app name> and will be dropped"